### PR TITLE
Add subsystem health

### PR DESCRIPTION
**Pull request by yaacov**
Add subsystem health

Requires kubevirt/web-ui#312

Ref: https://jira.coreos.com/browse/CNV-1151

Screenshots:
Peek 2019-04-17 15-17

### CODE REVIEW

**Comment by yaacov** [comment]
Ref: https://github.com/kubevirt/web-ui/pull/312

**Comment by yaacov** [comment]
Removeing WIP,
@suomiy, @rawagner ( although you wrote most of it :-) ) please review.
 
To @rawagner :tada: :cake: :beer: big thanks !

File: src/components/Dashboard/DashboardCard/DashboardCardTitleSeeAll.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+DashboardCardTitleSeeAll.defaultProps = {
+  children: null,
+};
+
+DashboardCardTitleSeeAll.propTypes = {
+  children: PropTypes.node,
+  title: PropTypes.string.isRequired,
+};
+
+export default DashboardCardTitleSeeAll;
```
Comment (rawagner) [inline src/components/Dashboard/DashboardCard/DashboardCardTitleSeeAll.js]: can you remove this default export and export DashboardCardTitleSeeAll itself ? we usually do not use default exports

File: src/components/SubsystemHealth/SubsystemHealth.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```
Comment (rawagner) [inline src/components/SubsystemHealth/SubsystemHealth.js]: SubsystemHealth component should go under Dashboard folder

File: src/components/Dashboard/DashboardCard/DashboardCardTitleSeeAll.js (hunk 1 above)
Comment (yaacov) [reply src/components/Dashboard/DashboardCard/DashboardCardTitleSeeAll.js]: Sure :+1:

File: src/components/SubsystemHealth/SubsystemHealth.js (hunk 2 above)
Comment (yaacov) [reply src/components/SubsystemHealth/SubsystemHealth.js]: :+1:

**Comment by yaacov** [comment]
moved subsystem into dashboard,

@rawagner please re review.

**Pull request merged**